src/settings/_colours-applied.scss

```diff
@@ -68,6 +68,15 @@ $govuk-secondary-text-colour: govuk-colour("grey-1") !default;
 
 $govuk-focus-colour: govuk-colour("yellow") !default;
 
+/// Focused text colour
+///
+/// Ensure that the contrast between the text and background colour passes
+/// WCAG Level AA contrast requirements.
```

Comment:
_Very_ minor, but there's a blank line between the description and any annotations elsewhere in this file, and I think generally through the codebase. Would be good to be consistent.